errai-ui/src/main/java/org/jboss/errai/ui/rebind/TranslationServiceGenerator.java

```diff
@@ -237,13 +236,13 @@ public String generate(TreeLogger logger, GeneratorContext context) {
     final Collection<URL> urls = new HashSet<URL>();
 
     addUrlsFromBundleAnnotations(bundleAnnotatedClasses, urls);
-    addUrlsFromErraiAppProperties(urls);
+    addUrlsFromErraiAppProperties( urls );

```

Comment:
Minor formatting issue: There's no space around method arguments in the Errai code style.